src/views/shared/base.njk

```diff
@@ -31,7 +31,7 @@
     {%- if pageErrorState %}
         {{ 'general.govuk.errorTitlePrefix' | translate }}
     {% endif %}
-    {%- if pageTitleKey %}{{ pageTitleKey | translateWithContextOrFallback(context) }} – GOV.UK{% endif %}
+    {%- if pageTitleKey %}{{ pageTitleKey | translateWithContextOrFallback(context) }} – GOV.UK One Login{% endif %}
```

Comment:
This ensures that all page titles end with `GOV.UK One Login` but it doesn't ensure that the title and the H1 are always the same. Do we need to do that too?

Reply:
Yep it's only needed for page titles not the headings

Reply:
Ah I see, I think they are almost always the same but I've just checked for the M2B screens and there is a discrepancy for `pyi-no-match` - the title is "Sorry, we could not confirm your details" while the header is "Sorry, we cannot confirm your identity". We should check this against the original designs/content doc

(There is probably a separate issue across the board here about whether we really need separate entries or could refactor to always use the heading in the title)

Reply:
(the Welsh is correct as is)